src/plugins/tables/tables.js

```diff
@@ -84,6 +84,29 @@ var componentName = "wb-tables",
 						dataTableExt = $.fn.dataTableExt;
 
 					/*
+					* Add printing support
+					*/
+					if ( $elm.hasAttribute( "print" ) ) {
+						var $btn = $( "<button class='btn btn-default mrgn-bttm-md' id='" + elmId + "_datatable-print'>Print Table</button>" );
+						$elm.before( $btn );
+						$btn.click( function() {
+							var css = "<link href='" + wb[ "/" ].replace( "js", "css" ) + "/theme" + wb.getMode() + ".css' rel='stylesheet' />", table = $( "<div>" ).append( $elm.clone() ).html(), html = css + table, printWin = window.open( "about:blank", "_blank" );
+							printWin.document.write( html );
+							printWin.document.close();
+							printWin.focus();
+							setTimeout( function() {
```

Comment:
Did you experienced that issue when you integrated your fix in wet? Can you look to use something more stable instead like by hooking on an event/callback. because why waiting 500 ms and not 1 tick or 10 ms or 3000 ms?